packages/lesswrong/components/jargon/GlossarySidebar.tsx

```diff
@@ -171,26 +217,18 @@ const GlossarySidebar = ({post, postGlossariesPinned, togglePin, showAllTerms, s
     return null;
   }
 
-  const tooltip = <div><p>Pin to highlight every instance of a term. (Opt/Alt + Shift + J)</p></div>;
-  const titleRow = currentUser ? (
-    <LWTooltip
-      title={tooltip}
-      inlineBlock={false}
-      placement='top-end'
-      popperClassName={classes.titleRowTooltipPopper}
-    >
-      <div className={classes.titleRow}>
-        <h3 className={classes.title}>
-          <strong>Glossary</strong>
-          <ForumIcon icon="Pin" className={classNames(classes.pinIcon, postGlossariesPinned && classes.pinnedPinIcon)} />
-        </h3>
-      </div>
-    </LWTooltip>
-  ) : (
-    <div className={classes.titleRow}>
-      <h3 className={classes.title}><strong>Glossary</strong></h3>
-    </div>
-  );
+  if (approvedTermsCount === 0 && unapprovedTermsCount === 0) {
```

Comment:
This was previously being handled by `displayTermCount` in `PostBody`.  If we're no longer going to use that, we should clean it up there